docs/dynamo.md

```diff
@@ -0,0 +1,90 @@
+## TorchDynamo(torch.compile) integration in PyTorch XLA
+
+Torchdynamo is a Python-level JIT compiler designed to make unmodified PyTorch programs faster. It provides a clean API for compiler backends to hook in and its biggest feature is to dynamically modify Python bytecode right before it is executed. In the pytorch/xla 2.0 release, PyTorch/XLA provided an experimental backend for the TorchDynamo for both inference and training. 
+
+The way that XLA bridge works is that Dynamo will provide a TorchFX graph when it recognizes a model pattern and PyTorch/XLA will use existing Lazy Tensor technology to compile the FX graph and return the compiled function. 
```

Comment:
can you add a few words on how a training cycle may consist of >1 graph - a contrast to the lazy execution flow.

for inference: we can make dynamo guarantee 1 graph to be generated if proper flag is set.

for training: today, we get at least 3 graphs.

Reply:
I do mentioned that below 
```
With the `torch.compile` you will see that PyTorch/XLA only traces the resent18 model once during the init time and executes the compiled binary everytime `dynamo_resnet18` is invoked
```
and
```
We expect to extract and execute 3 graphs per training step instead of one training step if you use the Lazy tensor.
```